### DISCUSSION TITLE
the player cannot be initialized

### DISCUSSION THREAD

**Discussion by kzwr** [vidstack/player#1361] [Q&A]
Hello Developer,

I followed the tutorial in the documentation (https://www.vidstack.io/docs/player/getting-started/installation/cdn?provider=hls&styling=default-theme), but the player cannot be initialized.

this is my code: `<html>

<head>
  <title>Video: Default Theme Example</title>
  <link rel="stylesheet" href="https://cdn.vidstack.io/player/theme.css" />
  <script src="https://cdn.vidstack.io/player.core" type="module"></script>
</head>

<body>
  <media-player title="Sprite Fight" src="https://files.vidstack.io/sprite-fight/hls/stream.m3u8">
    <media-provider></media-provider>
  </media-player>
</body>

</html>`

![20240724163909](https://github.com/user-attachments/assets/6660f6ed-7264-4d10-b490-88bdcb88dad9)

**Comment by kzwr** [comment]
![11](https://github.com/user-attachments/assets/9c37700c-0857-4524-a209-272f51e83707)

**Comment by mihar-22** [comment]
As explained in the docs, the option you've picked is for building your own layout. There is no UI included in this option. Choose the Default Layout if you'd like a ready solution.

**Reply by kzwr** [reply]
Thank you. I understand now that the option I picked is for building my own layout and does not include a UI. I will choose the Default Layout for a ready solution. The issue is resolved.

By the way, I want to commend you on the excellent work with this open-source project. 

It's very impressive and well-crafted.
